pkg-example-smf-bp/3gpp.yaml

```diff
@@ -0,0 +1,59 @@
+kind: ConfigMap
+metadata:
+  name: smf-configmap
+  labels:
+    app: free5gc
+data:
+  smfcfg.yaml: |
+    info:
+      version: 1.0.2
+      description: SMF initial local configuration
+
+    configuration:
+      serviceNameList:
+        - nsmf-pdusession
+        - nsmf-event-exposure
+        - nsmf-oam
+
+      sbi:
+        scheme: http
+        registerIPv4: smf-nsmf # IP used to register to NRF
+        bindingIPv4: 0.0.0.0  # IP used to bind the service
+        port: 80
+        tls:
+          key: config/TLS/smf.key
```

Comment:
Are these files? Where do they come from? I don't see them in the package. Presumably the workload cluster operator (which is generating the actual Deployment), will mount these from somewhere?

Reply:
This is coming done by the SMF operator. Either we create a specific configmap for SMF/AMF which need to be interpreted bu workload cluster. So this is a vendor specific CRD somehow. We use configmap for now.